src/Resources/views/CRUD/base_list_field.html.twig

```diff
@@ -35,7 +35,13 @@ file that was distributed with this source code.
         </a>
     {% else %}
         {% set isEditable = field_description.options.editable is defined and field_description.options.editable and admin.hasAccess('edit', object) %}
-        {% set xEditableType = field_description.type|sonata_xeditable_type %}
+        {% if isEditable==true and field_description.options.multiple is defined and field_description.options.multiple==true and value is iterable %}
```

Comment:
```suggestion
        {% if isEditable and field_description.options.multiple is defined and field_description.options.multiple==true and value is iterable %}
```

Reply:
Maybe we should use a single coding style?
https://github.com/sonata-project/SonataAdminBundle/search?l=Twig&q=%3D%3Dtrue

Reply:
For example, what about this condition?

https://github.com/sonata-project/SonataAdminBundle/blob/7619f20f25fea2432784c490539557a68424cf96/src/Resources/views/CRUD/list_choice.html.twig#L33